src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryFunctionsIT.java

```diff
@@ -62,6 +62,7 @@ protected void init() throws Exception {
         loadIndex(Index.ACCOUNT);
         loadIndex(Index.NESTED);
         loadIndex(Index.PHRASE);
+        loadIndex(Index.ONLINE);
```

Comment:
Any special reason to load this index? Because I don't see code changes on test cases in this IT.

Reply:
@dai-chen Sorry about the mistake. I put the ITs into QueryFunctionsIT at the first place, and moved them to JdbcTestIT but forgot to delete the loadIndex. Will update it, thanks!

Reply:
No worries. Thanks! 👍